### PR TITLE
[MAJOR] EDA-1353 Update match list method for serializer

### PR DESCRIPTION
**Pull request by vgatica-eb**
At beggining we changed **match_list** method in views_api.py file to accept a new dictionary as we mencioned on the acceptance criteria of TK EDA-1353.
While we were working, we discovered that we have a new big refactor to do, relate to accept more of 2 bots as players, so in **match_list** we implemented a for that can be iterated many times as number of tuples exist in the data list. So with this we resolved the problem of recieve more than 2 bots of server. But, we had to change the name of keys in for, to make it iterable as we said before --> bot_one **now is bot_1**. So, we had to modify class MatchSerializer, class Match and tests.

### CODE REVIEW

File: development/views_api.py [hunk 1]
```diff
@@ -6,15 +6,17 @@
 
 @csrf_exempt
 def match_list(request):
```
Comment (eb-cmoyano) [inline development/views_api.py:8]: Could you add a docstring for this API?

File: development/views_api.py [hunk 2]
```diff
@@ -6,15 +6,17 @@
...
-
     if request.method == 'POST':
-        data = JSONParser().parse(request)
+        req_data = JSONParser().parse(request)
+        data = {}
+        try:
+            data['game_id'] = req_data["game_id"]
+            for i, (name, score) in enumerate(req_data["data"], 1):
+                data[f'bot_{i}'] = name
+                data[f'score_p_{i}'] = score
```
Comment (eb-cmoyano) [inline development/views_api.py:16]: could you move this behavior to a single method/function/util in order to test it separately?

Comment (vgatica-eb) [reply development/views_api.py]: done in convert_data method

File: development/views_api.py (hunk 1 above)
Comment (vgatica-eb) [reply development/views_api.py:8]: done

**Review by ibrizuela-eb — APPROVED** [review]

**Pull request merged**